policies/slsa/verify-builder.rego

```diff
@@ -34,11 +38,12 @@ reason = v {
 	v := "builder mismatch"
 }
 
-violations = j {
-	j := {r |
-		input.evidence.predicate.buildDefinition.internalParameters.hostname != input.config.args.id
-		r = {
-			"builder_id": input.evidence.predicate.buildDefinition.internalParameters.hostname,
-		}
-	}
+violations = [{"id": input.evidence.predicate.buildDefinition.internalParameters.run_id}] if {
+	input.evidence.predicate.buildDefinition.internalParameters.context_type == "github"
```

Comment:
why specifically for github.
I think == "local" should return violation (bulid should not be local), anything else, meaning 
github, circle, azure .. etc is considered a CI builder and you can compare the run_id (they all should have it populated)